diesel/src/mysql/connection/bind.rs

```diff
@@ -9,6 +9,7 @@ pub struct Binds {
     lengths: Vec<libc::c_ulong>,
     is_nulls: Vec<ffi::my_bool>,
     mysql_binds: Vec<ffi::MYSQL_BIND>,
+    errors: Option<Vec<ffi::my_bool>>,
```

Comment:
I'm going to explore this, but if I end up going that way it'll be a separate PR as this one is large enough as it is.